clang/test/dpct/helper_files_ref/include/sparse_utils.hpp

```diff
@@ -0,0 +1,46 @@
+//==---- sparse_utils.hpp -------------------------*- C++ -*----------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __DPCT_SPARSE_UTILS_HPP__
+#define __DPCT_SPARSE_UTILS_HPP__
+
+#include <oneapi/mkl.hpp>
+#include <sycl/sycl.hpp>
+
+namespace dpct {
+namespace sparse {
+/// Describes properties of a sparse matrix.
+/// The properties are matrix type, diag, uplo and index base.
+class sparse_matrix_info {
```

Comment:
```suggestion
class matrix_info {
```

I'm thinking, since sparse is already namespace name, how about not repeat it in the class name to make the API more concise?

Reply:
good idea so it is sparse::matrix_info

Reply:
fixed